### ISSUE TITLE
simplify to code initializer rather than ctor

### ISSUE THREAD

**Issue by DickBaker** [dotnet/docs#41748]
### Type of issue

Outdated article

### Description

imho the first "**public class Taxi**" example should be simplified. The existing sample
```
public class Taxi
{
    public bool IsInitialized;

    public Taxi()
    {
        IsInitialized = true;
    }
}
```
could be
```
public class Taxi
{
    public bool IsInitialized = true;
}
```

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/programming-guide/classes-and-structs/using-constructors.md

### Document Version Independent Id

36d0bf3a-29bf-5162-c700-142b03f9a16c

### Article author

@BillWagner

### Metadata

* ID: 374a58d4-1905-c664-04b6-fd68e617f8c9 
* Service: **dotnet-csharp**
* Sub-service: **fundamentals**

---
[Associated WorkItem - 350760](https://dev.azure.com/msft-skilling/Content/_workitems/edit/350760)

**Comment by BillWagner** [comment]
There's a number of updates that could be made in the area of constructors. Everything from field initializers to `init` only properties to `required` properties and so on.

I agree that this example isn't a great one. Code like this should use an initializer instead. It should be part of a suite of examples that show the different techniques, and some guidance about determining which fits different design goals best.

**Comment by DickBaker** [comment]
yup this is doubtless one of MANY, but suggest we [MS+Community] try to improve the doc experience whenever we can and not get daunted by the enormity of the beast, so I hope you might accept this minor contrib.
Sometimes I get riled by something that results in a major work (Issue #10110, PR #10113) but hope these all contribute positive improvement to the doc real-estate. I hope that you welcome such assistance anyway!
